store/index.ts

```diff
@@ -27,8 +25,15 @@ export const mapRefAtom = atom<MapRef | null>(null);
 export const useMap = () => useAtomValue(mapRefAtom);
 export const setMap = () => useSetAtom(mapRefAtom);
 
-// Sneaky helpers
+/** Map Layer Configuration */
+// I see this getting really out of hand if the number of layers were to scale...
+export const activeLayersAtom = atom<any>({
+  daily: true,
+  weekly: false,
+});
 
```

Comment:
Before we add layers for different magnitudes we really need to engineer a better data structure for this atom. 

So it be one atom `get`ting the value of several other?